script-test/playbackstrategies/msestrategytest.js

```diff
@@ -812,7 +821,7 @@ require(
           };
 
           setUpMSE();
-          mockDashInstance.getBitrateInfoListFor.and.returnValue([{ bitrate: 1000 }, { bitrate: 2048 }, { bitrate: 3000 }]);
+          mockDashInstance.getBitrateInfoListFor.and.returnValue([{ bitrate: 1024000 }, { bitrate: 2000000 }, { bitrate: 3000000 }]);
```

Comment:
Is this this test case still true? Possibly misunderstanding, but if the audio bitrate changes within a video, would we not want to call the plugin, given the client is reporting combined bitrate now?

Reply:
This is still valid, I changed the mock values to represent something more reasonable. The event is called once, mediaKinds is video and both the video and audio bitrates are queried and combined. Rather than make more work for myself with the tests I just changed what was required.

Reply:
Ah, I'm still not understanding. The name of the test is:
"_should not call plugins with audio playback bitrate when mediaKind is video_"
But now we do care when the audio playback bitrate changes right? So why would we not call the plugins with this updated information?